runtime/src/lib.rs

```diff
@@ -40,8 +40,6 @@ mod standalone;
 
 #[cfg(feature = "standalone")]
 use standalone::*;
-// #[cfg(not(feature = "standalone"))]
-// use parachain::*;
```

Comment:
If we uncomment it, it still isn't used so that is why I removed it. If we know we're going to use it in the future, why not just add it back then. Imports do not require backwards compatibility.

Reply:
I added it back, was wrong about the CI